domino-ui/src/main/java/org/dominokit/domino/ui/lists/ListItem.java

```diff
@@ -24,7 +25,17 @@
     private HTMLAnchorElement element= a().css(ListStyles.LIST_GROUP_ITEM).asElement();
 
     public ListItem(T value) {
+    	element.setAttribute("tabindex", "0");
```

Comment:
@rjeeb It's for keyboard navigation. Currently you can't use/control list items with the keyboard because you can't focus or tab it. The accessability recommendation is to set tabindex to value "0" for elements that are none focussable per default but should be focussable for interaction. The whole PR should improve the accessability.

Reply:
I checked the recommendations and it says that the value should not be greater than 0, so this is good :+1: